docker/config/local_dev.env

```diff
@@ -29,12 +29,7 @@ resource.postgresql.database_port=5432
 secrets.postgresql.database_username=postgres
 secrets.postgresql.database_password=aPassword
 
-# resource.postgresql.database_replication_hostname=
-# secrets.postgresql.database_replication_password=
-# secrets.postgresql.database_replication_username=
-
-# postgres things for alembic
-# FIXME - sqlalchemy.url=postgresql://postgres:aPassword@postgresql:5432/socorro_migration_test
```

Comment:
We need to set this here so that migrations work in the local dev environment.

When the tests run, they override this value with the appropriate value for migration testing, so this works fine and doesn't affect anything else.